### DISCUSSION TITLE
Unnecessary Port Binding for so-steno

### DISCUSSION THREAD

**Discussion by Masaya-A** [Security-Onion-Solutions/securityonion#5979] [Unsupported Versions] [answered]
so-steno container gives a following error to /var/log/messages every 15mins.
`Oct 22 04:08:24  dockerd: time="2021-10-22T04:08:24.065899058Z" level=warning msg="Published ports are discarded when using host network mode"`
The reason is that port binding is written in a salt/pcap/init.sls even though network mode is host.
https://github.com/Security-Onion-Solutions/securityonion/blob/cfaa0e679c333f0671aca7bb058d46b22faee965/salt/pcap/init.sls#L120-L121

It seems no problem on some of my CentOS-standalone-environment when those lines are deleted.
But now I don't have enough time to test on Ubuntu or other deployment types.
Could you check and then delete those lines if no problem, please?

Security Onion version: 2.3.80
Install source: Network
OS: CentOS 7
Install type: Standalone

**Comment by dougburks** [comment, marked as answer]
We've created an issue for this:
https://github.com/Security-Onion-Solutions/securityonion/issues/5981
